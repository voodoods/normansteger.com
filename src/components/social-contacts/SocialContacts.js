import React, { Component } from 'react'
import StyledSocialContacts from './SocialContacts.styled'
import 'font-awesome/css/font-awesome.min.css'

class SocialContacts extends Component {
  render() {
    return (
      <StyledSocialContacts className="SocialContacts">
        <li>
          <a href="https://twitter.com/voodoods" target="_blank" rel="noopener noreferrer">
            <i className="fa fa-twitter-square fa-2x" />
          </a>
        </li>
        <li>
          <a href="https://www.xing.com/profile/Norman_Steger/cv" target="_blank" rel="noopener noreferrer">
            <i className="fa fa-xing-square fa-2x" />
          </a>
        </li>
        <li>
          <a href="https://www.linkedin.com/in/norman-steger-70568284/" target="_blank" rel="noopener noreferrer">
            <i className="fa fa-linkedin-square fa-2x" />
          </a>
        </li>
        <li>
          <a href="https://github.com/voodoods" target="_blank" rel="noopener noreferrer">
            <i className="fa fa-github-square fa-2x" />
          </a>
        </li>
      </StyledSocialContacts>
    )
  }
}

export default SocialContacts
