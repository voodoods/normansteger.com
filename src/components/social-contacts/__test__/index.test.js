import React from 'react'
import { shallow } from 'enzyme'
import SocialContacts from '../'

describe('<SocialContacts /> component index', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<SocialContacts />)
    expect(wrapper).toMatchSnapshot()
  })
})
