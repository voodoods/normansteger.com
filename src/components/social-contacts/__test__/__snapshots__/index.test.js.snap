// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialContacts /> component index should render without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SocialContacts />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <li>
          <a
            href="https://twitter.com/voodoods"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-twitter-square fa-2x"
            />
          </a>
        </li>,
        <li>
          <a
            href="https://www.xing.com/profile/Norman_Steger/cv"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-xing-square fa-2x"
            />
          </a>
        </li>,
        <li>
          <a
            href="https://www.linkedin.com/in/norman-steger-70568284/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-linkedin-square fa-2x"
            />
          </a>
        </li>,
        <li>
          <a
            href="https://github.com/voodoods"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-github-square fa-2x"
            />
          </a>
        </li>,
      ],
      "className": "SocialContacts",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="https://twitter.com/voodoods"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-twitter-square fa-2x"
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fa fa-twitter-square fa-2x"
            />,
            "href": "https://twitter.com/voodoods",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-twitter-square fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "a",
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="https://www.xing.com/profile/Norman_Steger/cv"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-xing-square fa-2x"
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fa fa-xing-square fa-2x"
            />,
            "href": "https://www.xing.com/profile/Norman_Steger/cv",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-xing-square fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "a",
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="https://www.linkedin.com/in/norman-steger-70568284/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-linkedin-square fa-2x"
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fa fa-linkedin-square fa-2x"
            />,
            "href": "https://www.linkedin.com/in/norman-steger-70568284/",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-linkedin-square fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "a",
        },
        "type": "li",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            href="https://github.com/voodoods"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              className="fa fa-github-square fa-2x"
            />
          </a>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fa fa-github-square fa-2x"
            />,
            "href": "https://github.com/voodoods",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-github-square fa-2x",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "a",
        },
        "type": "li",
      },
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  list-style: none;

  li {
    display: inline-block;
    margin-right: 5px;

    a {
      color: #c1ccd1;
    }
  }
",
        ],
      },
      "displayName": "styled.ul",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "ul",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <li>
            <a
              href="https://twitter.com/voodoods"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-twitter-square fa-2x"
              />
            </a>
          </li>,
          <li>
            <a
              href="https://www.xing.com/profile/Norman_Steger/cv"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-xing-square fa-2x"
              />
            </a>
          </li>,
          <li>
            <a
              href="https://www.linkedin.com/in/norman-steger-70568284/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-linkedin-square fa-2x"
              />
            </a>
          </li>,
          <li>
            <a
              href="https://github.com/voodoods"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-github-square fa-2x"
              />
            </a>
          </li>,
        ],
        "className": "SocialContacts",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="https://twitter.com/voodoods"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-twitter-square fa-2x"
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fa fa-twitter-square fa-2x"
              />,
              "href": "https://twitter.com/voodoods",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-twitter-square fa-2x",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="https://www.xing.com/profile/Norman_Steger/cv"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-xing-square fa-2x"
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fa fa-xing-square fa-2x"
              />,
              "href": "https://www.xing.com/profile/Norman_Steger/cv",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-xing-square fa-2x",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="https://www.linkedin.com/in/norman-steger-70568284/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-linkedin-square fa-2x"
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fa fa-linkedin-square fa-2x"
              />,
              "href": "https://www.linkedin.com/in/norman-steger-70568284/",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-linkedin-square fa-2x",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              href="https://github.com/voodoods"
              rel="noopener noreferrer"
              target="_blank"
            >
              <i
                className="fa fa-github-square fa-2x"
              />
            </a>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <i
                className="fa fa-github-square fa-2x"
              />,
              "href": "https://github.com/voodoods",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-github-square fa-2x",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            "type": "a",
          },
          "type": "li",
        },
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "rules": Array [
            "
  list-style: none;

  li {
    display: inline-block;
    margin-right: 5px;

    a {
      color: #c1ccd1;
    }
  }
",
          ],
        },
        "displayName": "styled.ul",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "ul",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
